src/cosmosdb-continuous-backup/azext_cosmosdb_pitr/_help.py

```diff
@@ -0,0 +1,151 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['cosmosdb create'] = """
+type: command
+short-summary: Creates a new Azure Cosmos DB database account.
+parameters:
+  - name: --locations
```

Comment:
Two questions:
1. as your parameter name is `--locations`, I would expect that I could specify multiple locations here. Is it by design?
2. You `--locations` is a complex parameter with as least three properties, right? Could we flatten it and group all these flatten prameters?